cypress/e2e/nestedDialogs.cy.js

```diff
@@ -13,15 +13,15 @@ describe('Nested dialogs', () => {
   })
 
   it('should close only the top most dialog when pressing ESC', () => {
-    cy.get('body').trigger('keydown', { keyCode: 27, which: 27 })
+    cy.get('html').trigger('keydown', { key: 'Escape', keyCode: 27, which: 27 })
     cy.get('[data-a11y-dialog="dialog-2"]').then(shouldBeVisible)
     cy.get('[data-a11y-dialog="dialog-3"]').then(shouldBeHidden)
   })
 
   it('should close only the top most dialog when clicking the backdrop', () => {
-    cy.get('body').trigger('keydown', { keyCode: 27, which: 27 })
```

Comment:
I have no idea why we performed an additional <kbd>ESC</kbd> here, but this is wrong.